README.md

```diff
@@ -66,6 +69,16 @@ Using [yarn](https://yarnpkg.com):
 yarn add @fingerprintjs/fingerprintjs-pro-spa
 ```
 
+Using [pnpm](https://pnpm.js.org):
+
+```sh
+pnpm add @fingerprintjs/fingerprintjs-pro-spa
+```
+
+## Requirements
+
+- For TypeScript users: Typescript 4.5 or higher
+
 ## Getting Started
 
```

Comment:
"Support + Feedback" -> "Support and feedback" 

Consider aligning the copy of this section as well as Licence with our template https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc

Reply:
Done!